events/pdf-listener.js

```diff
@@ -0,0 +1,58 @@
+const { Events } = require('discord.js');
+const FormData = require('form-data');
+const { wikiApiKey } = require('../config.json');
+const axios = require('axios');
+
+module.exports = {
+    name: Events.MessageCreate,
+    async execute(message) {
+        if (message.channel.id === '1298058637121556612' && message.attachments.size > 0) { // Channel to listen in
```

Comment:
Setting a const to `'1298058637121556612'` and giving it a descriptive name would make this easier to read. Could leave it as a global variable underneath imports